runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -26,59 +26,56 @@ namespace misc {
     ssize_t b;
 
     Interval();
-    explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
```

Comment:
That's surprising. From my tests I found that the explicit constructor will be used for `IntervalSet s(1, 2)`.

Anyway, the idea is simple: use the c-tor for `size_t` only with unsigned values and the one with `ssize_t` for signed ones. That means usually that the one for unsigned values is used (since most values are unsigned) and only in specific cases take the signed version.